docs/spdx-tag-value-mapping.md

```diff
@@ -0,0 +1,151 @@
+
+*** For the Files Analyzed field (3.8), this must be set to `false` if the SPDX document does NOT include File element sections for the Files contained within the Package. If File sections are included, then this should be set to `true` (or the field can be omitted); however, in that case additional mandatory fields are required, notably the Package Verification Code (3.9).
+
+### Relationship
+
+Whenever a relationship is being expressed between two SPDX elements, the following fields should appear.
+
+Relationships do not need to appear grouped together in any particular location. They typically appear immediately following one of the SPDX elements they are describing.
+
+ §  | SPDX Field name | Mandatory? | Brief description    | Example
+----|-----------------|------------|----------------------|---------
+7.1 | Relationship    |    Yes     | express the relationship between two SPDX elements | `Relationship: SPDXRef-requests HAS_PREREQUISITE SPDXRef-urllib3`
+
+A list of the various Relationship types that are currently recognized is available in section 7.1.1 of [the SPDX specification](https://spdx.org/specifications).
+
+### Other License
+
+The following fields should appear once per non-standard license (i.e., a license that is *not* on the [SPDX License List](https://spdx.org/licenses/) described in the SPDX document. Each Other License section should start with the `LicenseID` field.
+
+This section is not required (and must not be included) for any license that *is* already on the SPDX License List.
+
+ §  | SPDX Field name   | Mandatory? | Brief description    | Example
+----|-------------------|------------|----------------------|---------
+6.1 | License Identifier|    Yes     | unique identifier for license in SPDX document | `LicenseID: LicenseRef-BSD-3-Clause-variant-1`
+6.2 | Extracted Text    |    Yes     | actual text of license | `ExtractedText: <text>Redistribution and use in source and binary forms...</text>`
+6.3 | License Name      |    Yes     | human-readable name for license | `LicenseName: BSD-3-Clause variant 1`
+
+## Mapping
+
+Given the SPDX fields described above, the following discusses how to map those SPDX fields to Tern's existing data model. Items that I've marked with *** will likely require some additional discussion.
+
+### Document Creation
+
+Include once at the beginning of the SPDX document.
+
+ §  | SPDX Field name | Tern data model reference | Comments
+----|-----------------|---------------------------|---------
+2.1 | SPDX Version    | N/A                       | will always be `SPDXVersion: SPDX-2.1`
+2.2 | Data License    | N/A                       | will always be `DataLicense: CC0-1.0`
+2.3 | SPDX Identifier | N/A                       | will always be `SPDXID: SPDXRef-DOCUMENT`
+2.4 | Document Name   | **TBD**                   | does Tern have a human-readable way to refer to the image or Dockerfile being analyzed?
```

Comment:
This is optional for Tern because, in most cases, the image is pulled from a container image registry with no associated Dockerfile. Would a image's repo and tag name do? Eg: "docker.io/debian:buster"?